browse/static/js/litmaps.js

```diff
@@ -0,0 +1,47 @@
+(function () {
+    const arxivId = window.location.pathname.split('/').reverse()[0];
```

Comment:
Hi @axtonpitt, I noticed when accessing a specific version of a paper (i.e. when the arXiv ID in the path has a version affix), Litmaps is less successful with the lookup. So I would exclude the version affix from all lookups. Example:

https://beta.arxiv.org/abs/2001.00001v2

You can get the canonical ID from one of the metatags as follows:
```suggestion
    const arxivId = document.head.querySelector("[name~=citation_arxiv_id][content]").content;
```